jetty-webapp/src/main/java/org/eclipse/jetty/webapp/JmxConfiguration.java

```diff
@@ -32,7 +32,7 @@
     public JmxConfiguration()
     {
         addDependents(WebXmlConfiguration.class, MetaInfConfiguration.class, WebInfConfiguration.class);
-        protectAndExpose("org.eclipse.jetty.jmx.");
+        protectAndExpose("org.eclipse.jetty.util.annotation", "org.eclipse.jetty.jmx.");
```

Comment:
This is correctly restoring the jetty-9 behaviour, so good to proceed with.

BUT ultimately we should avoid exposing jetty classes like this.  So we should consider an option of how we can use Jmx in the server and not expose to the webapp: either by making this line conditional or moving into another alternate optional configuration (which would need it's own jar).